src/lib/types/events.ts

```diff
@@ -338,14 +326,12 @@ export class FeatureEnvironmentEvent extends BaseEvent {
         featureName: string;
         environment: string;
         createdBy: string | IUser;
-        tags: ITag[];
```

Comment:
I think it's fine to leave them as optional just in case some of these are used in Enterprise repository, it **could** make the transition easier.

But it might be also we move faster by breaking the contract and fixing it quickly, maybe we can check this by linking your branch into enterprise and having the change ready

Reply:
I think this is definitely a case where we move faster by breaking the contract and fixing it quickly. I have the Enterprise PR ready to go if you'd like to have a look at it.